api/swagger.yml

```diff
@@ -1777,6 +1777,28 @@ paths:
         required: true
         schema:
           type: string
+    get:
+      deprecated: true
```

Comment:
What's the replacement?  Can we document the replacement -- even as a `description`?

Reply:
Added description part of the summary